### DISCUSSION TITLE
Command line for container execution in CI

### DISCUSSION THREAD

**Discussion by jgsuess** [oracle/macaron#337] [Q&A] [answered]
While the manual provides an invocation script that allows execution from a CLI to run macaron in docker, a common use case in CI is to run the tool inside the container, with the source mounted from the local repo.

How can I obtain the command-line that I need to invoke macaron when I am `inside` the container. In other words, what is the API/CLI for the application in the container?

**Comment by behnazh-w** [comment]
Inside the container you should be able to run `macaron` as a Python package. It takes the exact same arguments as the Docker container.

**Comment by jgsuess** [comment]
Is that the same parameter list as produced by `run-macaron.sh --help`? Or where do I find the container cli description?

**Comment by behnazh-w** [comment, marked as answer]
Yes, the parameters are the same as `run-macaron.sh --help`. You can also just run `macaron --help` inside the container.

**Reply by jgsuess** [reply]
Thank you. That unblocks me.
